packages/dx-react-grid-demos/src/demo-sources/grid-lazy-loading/remote-data.jsxt

```diff
@@ -18,6 +18,7 @@ const VIRTUAL_PAGE_SIZE = 100;
 const MAX_ROWS = 50000;
 const URL = 'https://js.devexpress.com/Demos/WidgetsGalleryDataService/api/Sales';
 const getRowId = row => row.Id;
+let lastQuery;
```

Comment:
It should be declared inside function

Reply:
If we will do it, every render it will be `undefined` (and we catch infnite loading).
We should declare it outside or use `useState`